blueprints/addon/files/.github/workflows/ci.yml

```diff
@@ -19,9 +19,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3<% if (pnpm) { %>
-      - uses: pnpm/action-setup@v2
+      - uses: pnpm/action-setup@v4
```

Comment:
is this the one that reads `packageManager`?

If so, maybe next we set packageManager and remove with version?

Reply:
@NullVoxPopuli yeah, fine by me as a follow up - so we do one thing in one PR